extension/src/test/suite/setup/index.test.ts

```diff
@@ -218,6 +218,7 @@ suite('Setup Test Suite', () => {
       setup.setCliCompatible(undefined)
```

Comment:
I'm seeing this error appear when I run the vscode tests, but I can't figure out what test is causing it:

```
dvc:cover-vscode-run: rejected promise not handled within 1 second: TypeError: vscode_1.workspace.getConfiguration(...).get is not a function
dvc:cover-vscode-run: stack trace: TypeError: vscode_1.workspace.getConfiguration(...).get is not a function
dvc:cover-vscode-run: 	at getConfigValue (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/vscode/config.js:518:48)
dvc:cover-vscode-run: 	at Config.getCliPath (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/config.js:2811:40)
dvc:cover-vscode-run: 	at Setup.<anonymous> (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7718:62)
dvc:cover-vscode-run: 	at Generator.next (<anonymous>)
dvc:cover-vscode-run: 	at /home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7275:125
dvc:cover-vscode-run: 	at new Promise (<anonymous>)
dvc:cover-vscode-run: 	at __awaiter (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7239:10)
dvc:cover-vscode-run: 	at Setup.getEnvDetails (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7716:12)
dvc:cover-vscode-run: 	at Setup.<anonymous> (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7766:67)
dvc:cover-vscode-run: 	at Generator.next (<anonymous>)
dvc:cover-vscode-run: 	at fulfilled (/home/runner/work/vscode-dvc/vscode-dvc/extension/dist/setup/index.js:7248:24)
```

Reply:
Did you figure it out? I cannot see it.

Reply:
Still haven't figured it out! Weird, you're not seeing it locally? You can see the error in the [test logs](https://github.com/iterative/vscode-dvc/actions/runs/4704508398/jobs/8344203528?pr=3688#step:7:679).

Reply:
Seem to be coming from that test: "should be able to handle a message to update the table depth".  Probably a stub missing or affecting some part it shouldn't.

Reply:
``` 
stub(workspace, 'getConfiguration').returns({
        update: mockUpdate
      } as unknown as WorkspaceConfiguration)
```
in test "should handle a message to set dvc.studio.shareExperimentsLive" does not have a get method. The stub is probably not restored after that.